### ISSUE TITLE
File Upload in IIS server

### ISSUE THREAD

**Issue by sridhar2392** [raben/Akamai#8]
I couldn't upload a file in IIS server but i can in localhost (wamp, xampp). Is there anything I'm missing ?

**Comment by talk2cerlin** [comment]
authentication is happening successfully? please check if the file upload is not happening after successful authentication.

**Comment by sridhar2392** [comment]
yes authentication is happening successfully. As i mentioned the file upload is happening like creating a file and writing text into it. Exactly which function should i use for a file upload?

**Comment by sridhar2392** [comment]
If i try to use print_r($service); after the authorization line,  i get the status as "0" for authorization. My credentials are correct. Is there any dependency that i should add for this code to work.

**Comment by talk2cerlin** [comment]
check your server time. even a 10 seconds difference would make your request invalid. make sure your server time is same as internet time. You can use some tools to sync the time

**Comment by sridhar2392** [comment]
The time is correct. I have verified it. This code is working in mine but if i try to run it in another system it is not working.

**Comment by sridhar2392** [comment]
<?php
require 'Akamai.php';
$service = new Akamai_Netstorage_Service('******-nsu.akamaihd.net');
$service->authorize('KL65******************************JI','keyname','5');
print_r($service);
$service->dir('/12345');
print_r($service);
?>
// file name- index.php
This is the code I've used. The Folder contains the package that is shared as a zip file [https://github.com/raben/Akamai].  
I've dowloaded the zip file-> extracted it -> included index.php -> and have run in wamp localhost. 

Not working. Need clarification about any extra dependency to be added or is there any steps that i missed ?

**Comment by sridhar2392** [comment]
It is working on localhost (wamp, xampp) but not in IIS. In WAMP i added curl ssl local certificate. Is there any certificate that i need to add in IIS server. ?
